docs/developing-contracts/codec.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_label: Encode/Decode data
+sidebar_position: 4
+---
+
+# Encode/Decode data
+
+Gear uses the `parity-scale-codec`, a Rust implementation of the SCALE Codec. SCALE is a lightweight format that allows encoding/decoding which makes it highly suitable for resource-constrained execution environments like blockchain runtimes and low-power/low-memory devices.
+
+To use scale codec in your program, you should add it in `Cargo.toml`:
+
+```toml
+[dependencies]
+
+// ...
+codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false, features = ["derive", "full"] }
+```
+
+```rust
+#[derive(Encode, Decode)]
+enum MyType {
+    MyStruct { field: ... },
+    ...
+}
+```
```

Comment:
Seems good to mention that we use Encode and Decode trait bound in some gstd methods for soft wrapping (it's difference between e.g. `send` and `send_bytes`)

Reply:
Left a note about the difference